src/components/ButtonMenu/readme.md

```diff
@@ -454,3 +454,254 @@ class ButtonMenuExample extends React.Component {
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato Light';
+    font-size: 24px;
+    text-align: left;
+    color: ${props => props.palette.text.main};
+`;
+
+const CardSubtitle = styled.h4.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 12px;
+    color: ${props => props.palette.text.header};
+    display: block;
+`;
+
+const Label = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato Bold;
+    font-size: 24px;
+    line-height: 24px;
+    color: ${props => props.blue? props.palette.brand.main : props.palette.text.main};
+    text-transform: uppercase;
+`;
+
+const Description = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.disabled};
+`;
+
+const ColumnContainer = styled.div.attrs(props => props.theme.rainbow)`
+    min-height: 92px;
+    border-left: ${props => props.noBorder? "0":"1"}px solid ${props => props.palette.text.disabled};
+    ${props => !!props.flex ? 'flex: ' + props.flex + ';' :(props.noBorder? "width: 49%;":"width: 17%;")}
+    @media (max-width: 700px) {
+        border-left: none;
+        width: 100%;
+    }
+`;
+
+const Column = props => {
+    const { label, description, specialColor, noBorder, children, flex } = props;
+
+    return (
+        <ColumnContainer
+            className="rainbow-flex rainbow-flex_column rainbow-justify_center rainbow-align_center"
+            noBorder={!!noBorder}
+            flex={flex}
+        >
+            {children}
+            <Label blue={!!specialColor}>
+                {label}
+            </Label>
+            <Description>
+                {description}
+            </Description>
+        </ColumnContainer>
+    );
+}
+
+const RowContainer = styled.div`
+    min-height: 92px;
+    width: 49%;
+    @media (max-width: 700px) {
+        width: 100%;
+    }
+`;
+
+const Row = props => {
+    const { children } = props;
+    return (
+        <RowContainer className="rainbow-flex rainbow-justify_space-around">
+            {children}
+        </RowContainer>
+    );
+};
+
+const Point = styled.span.attrs(props => props.theme.rainbow)`
+    height: 10px;
+    width: 10px;
+    background-color: ${props => props.palette.brand.main};
+    border-radius: 50%;
+    display: inline-block;
+`;
+
+const Line = styled.hr.attrs(props => props.theme.rainbow)`
+    border: 1px solid ${props => props.palette.brand.main};
+    background-color: ${props => props.palette.brand.main};
+    width: 98%;
+    padding: 0;
+    margin: 0;
+`;
+
+const SegmentContainer = styled.div`
+    width: 100%;
+    height: 20px;
+`;
+
+const Segment = props => {
+    return (
+        <SegmentContainer className="rainbow-flex rainbow-align_center">
+            <Point />
+            <Line />  
+            <Point />  
+        </SegmentContainer>
+    );
+}
+
+function getdDateDiference ( dateIni, dateEnd ) {
```

Comment:
getdDateDiference  => getdDateDifference
I think it is better to use startDate instead of dateIni